src/Swarm/Game/World.hs

```diff
@@ -225,31 +241,43 @@ lookupTerrain i (World f t _) =
   ((U.! tileOffset i) . fst <$> M.lookup (tileCoords i) t)
     ? fst (runWF f i)
 
--- | A stateful variant of 'lookupTerrain', which first loads the tile
+-- | A stateful variant of "lookupTerrain", which first loads the tile
```

Comment:
Note the single quotes are semantically meaningful --- they cause Haddock to turn the quoted thing into a link to the named function or type.